k8s/namespaces/ccd/ccd-admin-web/aat.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-admin-web
+spec:
+  releaseName: ccd-admin-web
+  values:
+    nodejs:
+      environment:
+        ADMINWEB_ELASTIC_INDEX_URL: http://ccd-definition-store-api-aat.service.core-compute-aat.internal/elastic-support/index
```

Comment:
you might need to add this to aat `kustomization.yaml` as well  (under patchesStrategicMerge)

Reply:
Well, you're right that if you want aat.yaml to be loaded this needs adding.

But this is just setting it to the same value that's in the chart:
https://github.com/hmcts/ccd-admin-web/blob/ab7b4bf287a802657818a44c9be16e44e533f2bb/charts/ccd-admin-web/values.yaml#L48

This file should be removed or just have a dummy var if you want to be able to easily update env values, and if you keep the file you need to add it to `patchesStrategicMerge`

Reply:
Was keeping it simple, ok to delete @MSancaktutar ?

Reply:
Deleted the file.